ci/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM ubuntu:18.04
```

Comment:
It would be best to use the redhat UBI images

Reply:
Could start with https://catalog.redhat.com/software/containers/ubi8/ubi/5c359854d70cc534b3a3784e and then install python/node/etc. This image is always most up to date in the past when it comes to patching vulnerabilities